src/main/webapp/WEB-INF/views/newOrderAll.jsp

```diff
@@ -0,0 +1,65 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html lang="pl">
+<%@ include file="header.jsp" %>
```

Comment:
Polecam https://www.baeldung.com/thymeleaf-in-spring-mvc aby nie wrzucać w każdy widok nagłówków